.github/workflows/test.yaml

```diff
@@ -8,7 +8,6 @@ jobs:
     strategy:
       matrix:
         php-versions: ['7.3', '7.4', '8.0']
-        package-stability: ['stable', 'alpha']
```

Comment:
Can you explain why you are having removed this line? 

In future, we may use it - when we will have an alpha version of the lib.

Reply:
Due to the fact that we have a setting "minimum-stability": "alpha" in composer.json, we will not get 6 tests, but only 3. The tests for alpha and stable will merge together.

Reply:
We do not have minimums-stability in Parser package. Or I'm missing somethings

Reply:
No, there is. In this package (xparse/parser). Please check again.

Reply:
Oh, my bad. Looks good. 👍

Reply:
And we cannot remove it. Tests without this config: https://github.com/saintdron/Parser/actions/runs/789349991